packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -1,12 +1,7 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
Finally this file is not 1500+ lines anymore! Thanks for making this one smaller